src/main/java/core/basesyntax/strategy/operations/BalanceOperation.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.strategy.operations;
+
+public class BalanceOperation implements DailyOperationHandler {
+    @Override
+    public int getQuantity(int quantity) {
```

Comment:
```suggestion
    public void apply(int quantity) {
```
цей метод може оновлювати значення в сторейджі відповідно до типу операціх